app/controllers/resources_controller.rb

```diff
@@ -38,8 +38,46 @@ def suggestions
     render json: ToolFilterService.new(params).call, include: params[:include], fields: field_params, status: :ok
   end
 
+  def publish_translation
```

Comment:
This is not behind authorization currently, it should be added to the `:authorize` `before_action`